plugins/doc_fragments/powerstore.py

```diff
@@ -49,8 +49,10 @@ class ModuleDocFragment(object):
               - If not passed, it will take 443 as default.
           type: int
     requirements:
-      - A Dell PowerStore storage system version 3.0.0.0 or later.
+      - A Dell PowerStore storage system version 3.6.0.0 or later.
       - PyPowerStore 3.4.1.
+      - Ansible-core 2.17 or later.
```

Comment:
I guess Either this we can revert to exiting one because modules are complaint with ansible and python versions. or write generic lines like Ansible-core, SDK version/Array version accordingly so we need to update in each release. Also, we have readme where we do mention about all supported versions.
I feel we need to address this thig on our other collection as well.

Reply:
+ @forrestxia , @gokul-srivathsan , @vangork do you have any thoughts on this?

Reply:
Talked with @forrestxia , @vangork  and the team. We will remove ansible and python from module doc, keep array version and sdk.
Please correct me if I misunderstood, @forrestxia , @vangork

Reply:
By refering to azure doc of https://docs.ansible.com/ansible/latest/collections/azure/azcollection/azure_rm_resourcegroup_module.html#requirements, can we just have the python version and full installtion instruction redirection to https://github.com/dell/ansible-powerstore/blob/main/docs/INSTALLATION.md?
Dose it make sense? @baoy1 @Bhavneet-Sharma @forrestxia